src/styles/card.scss

```
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
```

Values should be multiples of 4. How about 
```suggestion
  padding: 16px 12px;
```
OR

```suggestion
  padding: 16px;
```